src/components/BeverageForm.js

```diff
@@ -29,25 +30,28 @@ type Props = {|
   submitButtonLabel: string,
 |};
 
-type InjectedProps = FormProps;
+type InjectedProps = {
+  ...FormProps,
+  availabilityStore: DAOEntityStore<Availability>,
+  glassStore: DAOEntityStore<Glass>,
+  srmStore: DAOEntityStore<Srm>,
+  styleStore: DAOEntityStore<Style>,
+};
 
 @form()
+@withDAOEntityStore('availabilityStore', DAOApi.AvailabilityDAO)
+@withDAOEntityStore('srmStore', DAOApi.SrmDAO)
+@withDAOEntityStore('glassStore', DAOApi.GlassDAO)
+@withDAOEntityStore('styleStore', DAOApi.StyleDAO)
 @observer
 class BeverageForm extends InjectedComponent<InjectedProps, Props> {
-  _availabilityStore: DAOEntityStore<Availability> = new DAOEntityStore(
-    DAOApi.AvailabilityDAO,
-  );
-  _srmStore: DAOEntityStore<Srm> = new DAOEntityStore(DAOApi.SrmDAO);
-  _glassStore: DAOEntityStore<Glass> = new DAOEntityStore(DAOApi.GlassDAO);
-  _styleStore: DAOEntityStore<Style> = new DAOEntityStore(DAOApi.StyleDAO);
-
   componentWillMount() {
-    this._availabilityStore.fetchMany();
-    this._srmStore.fetchMany({
+    DAOApi.AvailabilityDAO.fetchMany();
```

Comment:
You shouldn't have to make these calls here. I'd prefer that the store call this function when you use the `allItems` getter